po/uk.po

```diff
@@ -7998,7 +7998,9 @@ msgstr "Словник перекладених повідомлень"
 msgid ""
 "\n"
 "ID Views\n"
+"\n"
```

Comment:
The messages to be translated get autogenerated using the common [gettext](https://docs.python.org/2/library/gettext.html) practice, therefore it does not make sense changing it here but should be changed in the code if there's a need for it.